src/Scientist/IExperiment.cs

```diff
@@ -36,6 +36,12 @@ public interface IExperiment<T>
         /// Defines a custom func used to compare results.
         /// </summary>
         void Compare(Func<T, T, bool> comparison);
+
+        /// <summary>
+        /// Defines the check to run to determine if mismatches should be ignored.
+        /// </summary>
+        /// <param name="block">The delegate to execute</param>
+        void Ignore(Func<bool> block);
```

Comment:
Shouldn't Ignore take a `Func<Observation<T>, Observation<T>, bool>` or `Func<T, T, bool>?  It looks like the ruby framework will submit a call that passes the control and candidates values in, and returns back the boolean result.

https://github.com/github/scientist/blob/master/lib/scientist/experiment.rb#L153